test/battle/ability/aerilate.c

```diff
@@ -45,12 +45,134 @@ SINGLE_BATTLE_TEST("Aerilate can not turn certain moves into Flying type moves")
     }
 }
 
-TO_DO_BATTLE_TEST("Aerilate boosts power of affected moves by 20% (Gen7+)");
-TO_DO_BATTLE_TEST("Aerilate boosts power of affected moves by 30% (Gen6)");
+SINGLE_BATTLE_TEST("Aerilate boosts power of affected moves by 20% (Gen7+) or 30% (Gen6-)", s16 damage)
```

Comment:
I tend to avoid "GenX-", instead I do "Gen1-X"
```suggestion
SINGLE_BATTLE_TEST("Aerilate boosts power of affected moves by 20% (Gen7+) or 30% (Gen1-6)", s16 damage)
```

Reply:
Done :)